src/commands/AttendanceCommand.ts

```diff
@@ -90,12 +89,22 @@ export class AttendanceCommand implements ICommand {
             const bs = this.conference.getAuditoriumBackstage(await auditorium.getId());
             const inviteTargets = await this.conference.getInviteTargetsForAuditorium(auditorium);
             const bsInviteTargets = await this.conference.getInviteTargetsForAuditorium(auditorium, true);
-            await append(inviteTargets, bsInviteTargets, await auditorium.getId(), auditorium.roomId, bs.roomId, doAppend);
+            try {
+                await append(inviteTargets, bsInviteTargets, await auditorium.getId(), auditorium.roomId, bs.roomId, doAppend);
+            }
+            catch (error) {
+                throw new Error(`Error calculating invite acceptance in auditorium ${auditorium}: ${error.toString()}`)
+            }
         }
         for (const spiRoom of this.conference.storedInterestRooms) {
             const doAppend = !!targetAudId && (targetAudId === "all" || targetAudId === await spiRoom.getId());
             const inviteTargets = await this.conference.getInviteTargetsForInterest(spiRoom);
-            await append(inviteTargets, null, await spiRoom.getId(), spiRoom.roomId, null, doAppend);
+            try {
+                await append(inviteTargets, null, await spiRoom.getId(), spiRoom.roomId, null, doAppend);
+            }
+            catch (error) {
+                throw new Error(`Error calculating invite acceptance in special interest room ${spiRoom}: ${error.toString()}`)
```

Comment:
```suggestion
                throw new Error(`Error calculating invite acceptance in special interest room ${spiRoom}`, {cause: error });
```

Gives you a nice stacktrack

```
> new Error('Oh no, an error', { cause: new Error('caused by this cheeky thing')})
Error: Oh no, an error
    at REPL2:1:1
    at Script.runInThisContext (node:vm:121:12)
    ... 7 lines matching cause stack trace ...
    at [_line] [as _line] (node:internal/readline/interface:887:18) {
  [cause]: Error: caused by this cheeky thing
      at REPL2:1:39
      at Script.runInThisContext (node:vm:121:12)
      at REPLServer.defaultEval (node:repl:593:29)
      at bound (node:domain:432:15)
      at REPLServer.runBound [as eval] (node:domain:443:12)
      at REPLServer.onLine (node:repl:923:10)
      at REPLServer.emit (node:events:526:35)
      at REPLServer.emit (node:domain:488:12)
      at [_onLine] [as _onLine] (node:internal/readline/interface:416:12)
      at [_line] [as _line] (node:internal/readline/interface:887:18)
}
```

This feature is not used nearly enough https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error#cause

Reply:
Cool thanks for the insight - I am new to typescript/javascript so appreciate the tips :)